Fixed: `highlights_for` would fail for books with large numbers of highlights

phillco opened the pull request:
This endpoint likes to throw 500 exceptions for books with large numbers of highlights (~700 in my experience). Some testing reveals batches over 250 or so to be unreliable, so I chose 200 as the default batch size.

Also theoretically support > 1000 highlights per book.

speric commented on lib/kindle_highlights/client.rb:
```diff
@@ -12,9 +15,20 @@ def initialize(email_address, password)
+      cursor = 0
+      highlights = []
+
+      while true
```
Do you think it'd be more idiomatic to do:

``` ruby
loop do
  # get items
  break unless json['items'] and json['items'].length > 0
  # move cursor, etc.
end
```

phillco replied:
Nice! I don't actually write Ruby all that often :D. I'll make the change.

speric commented:
Thanks for opening this PR! I left a small bit of feedback.

speric commented:
@phillco would you mind squashing these commits down? Thanks!

phillco committed:
Fixed: `highlights_for` would fail for books with large numbers of highlights

This endpoint likes to throw 500 exceptions for books with large numbers of highlights (~700 in my experience). Some testing reveals batches over 250 or so to be unreliable, so I chose 200 as the default batch size.

Also theoretically support > 1000 highlights per book.

phillco commented:
@speric done, sorry for the delay!
